airbyte-integrations/connectors/source-snowflake/source_snowflake/schema_builder.py

```diff
@@ -191,7 +191,9 @@ def format_field(field_value, field_type, local_time_zone_offset_hours=None):
 
             if airbyte_format == 'time' and airbyte_type == 'time_without_timezone':
                 ts = float(field_value)
-                dt = datetime.fromtimestamp(ts)
+                unix_epoch = datetime(year=1970, month=1, day=1)
+                delta_seconds_offset = timedelta(seconds=ts)
+                dt = unix_epoch + delta_seconds_offset
                 return dt.strftime("%H:%M:%S")
 
         except ValueError:
```

Comment:
Regarding the int conversion (at the time of this comment, it is line 204), why do we parse as a string before parsing as an int?

Reply:
removed